gpuflow/dataplane/gpu/cuda/cuda_lpm_factory.cu

```diff
@@ -160,7 +160,8 @@ int IPv6LPMFactory::AddLPMRule(uint8_t *ipv6_address, uint8_t depth, uint8_t nex
 
   }
 
-  SetupIPv6RuleEntry<<<1, end -start>>>(IPv6TBL24, start, next_hop, depth);
+  // BUG: When (end-start) > 1024, that is, when depth is smaller than 14, SetupIPv6RuleEntry does not work. 
```

Comment:
It may due to the limit number of threads? Extend this kernel function into block + thread indexes. Like `InitLPM` kernel function did.

Reply:
Or you can describe in detailed for the abnormal result.

Reply:
Or if you are not going to fix this, use `FIXME` for highlighting in IDE.
```c++
// FIXME: bug description
```

Reply:
Advice: 
[operator precedence in c++](http://en.cppreference.com/w/cpp/language/operator_precedence). If you are not sure the precedence is, just add braces. At some points, it's better for code readability, I think.